ui/tests/index.js

```diff
@@ -1,5 +1,9 @@
 import Vue from 'vue';
 import Vuetify from 'vuetify';
 
+import SnackbarError from '@/components/snackbar/SnackbarError';
+
+Vue.component('SnackbarError', SnackbarError);
+
```

Comment:
This commit should be included in the previous one. Please make sure tests are running on all commits.

Reply:
Should I fixup the last two commits?

Reply:
Yes.